extensions/podman/packages/extension/src/extension.ts

```diff
@@ -108,6 +112,21 @@ const setupPodmanNotification: extensionApi.NotificationOptions = {
 };
 let notificationDisposable: extensionApi.Disposable;
 
+// Alert for running podman-mac-helper
+// Add notification that podman-mac-helper needs setting up
+let doNotShouldMacHelperSetup = false;
+const setupMacHelperNotification: extensionApi.NotificationOptions = {
```

Comment:
feels like a constants, feels like something that should be in a dedicated file 🌺